sigs/sig-scalability/README.md

```diff
@@ -0,0 +1,19 @@
+# Argo SIG Scalability
+
+Argo SIG Scalability is focused on the scalability and performance of Argo CD project.
```

Comment:
Given that the proposal leading the creation of this SIG is focused on Argo CD, I've included that in the description. If the scope changes in the future, we can update the SIG.

Though this leads me to consider calling this the `Argo CD SIG Scalability`, I'm unsure if that is correct.

Reply:
I think we can call it "Argo SIG Scalability" and just state initial focus of the SIG is ArgoCD only and the scope described here. If there is a case that there are folks that want to join and focus on something else like Workflows, they can go ahead do a proposal to expand the SIG scope but no need for a new SIG.

Reply:
i totally agree with @csantanapr 's suggestion. ideally this should grow beyond CD and we can step up to help with other argo projects as well.

Reply:
[Done](https://github.com/argoproj/argoproj/pull/192/commits/616cb1583dc886881d6ff8c029c5d869c64ce4d7)!